doc/release-notes/0.10/0.10.md

```diff
@@ -0,0 +1,187 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.10.0 release notes
+
+These release notes support the [Eclipse OpenJ9 0.10.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.10.0/plan).
+
+
+
+## Binaries and supported environments
+
+This release adds compatibility with OpenJDK v11. We expect a binary of OpenJDK 11 with OpenJ9 to be available from the AdoptOpenJDK community in the near future at the following link:
```

Comment:
Seems this statement should be updated since the link is now active. Also I'm wondering if this should link to the actual release page when it is created.